### ISSUE TITLE
findDangerousChanges doesn't work as expected after lexicographic sort and deserialization

### ISSUE THREAD

**Issue by mrtnzlml** [graphql/graphql-js#2150]
Hello! My colleague reported a bug where `findDangerousChanges` reports invalid dangerous change after serializing the schema with `lexicographicSortSchema` and then deserializing it again. I tried to reproduce the error here: https://github.com/kiwicom/graphql-bc-checker/commit/f5fa6be30ab5f0569e6418f75f9425f724a0b7a1#diff-b3245f2dbca2b4c14d7bc1ff7eef9c87

Note that we are using one schema which has unsorted args, then printing it into a string and later trying to compare this string (potentially saved on disk) with the actual schema. It should not report any dangerous changes but it does report this:

```json
{
  "description": "RootQueryType.xxx arg yyy has changed defaultValue from {aaa: \\"aaa\\", bbb: \\"bbb\\"} to {bbb: \\"bbb\\", aaa: \\"aaa\\"}.",
  "type": "ARG_DEFAULT_VALUE_CHANGE",
}
```

Thanks for checking it! :)

**Comment by Neitsch** [comment]
Yep, this looks like a true issue. For input type default values we apparently just compare it's stringified value instead of doing a deep comparison. Could you maybe lexicographically sort the other schema as well before doing the comparison (as a temporary workaround).
The proper fix we should probably put in place is to lexicographically sort the value we stringify.

**Comment by IvanGoncharov** [comment]
@mrtnzlml @Neitsch Fixed in #2151 if you have time can you please review it?
